modules/cart/src/Plugin/views/field/EmptyCartButton.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace Drupal\commerce_cart\Plugin\views\field;
+
+use Drupal\commerce_cart\CartManagerInterface;
+use Drupal\commerce_order\Entity\Order;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\views\Plugin\views\field\FieldPluginBase;
+use Drupal\views\Plugin\views\field\UncacheableFieldHandlerTrait;
+use Drupal\views\ResultRow;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Defines a form element for emptying the cart.
+ *
+ * @ViewsField("commerce_order_item_empty_cart_button")
+ */
+class EmptyCartButton extends FieldPluginBase {
```

Comment:
I assumed this would just be an optional item a user could add to their Cart Form View and wasn't sure if tests got done in those cases. 

So are you saying it should be changed to be an item that's enabled by default, or keep it disabled by default and the test should enable this item for the View, then test it?